hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceConfigurationBuilder.java

```diff
@@ -455,12 +475,13 @@ private static void determineMetricSetPrometheus(ModelNode config,
 
         ModelNode platformValueNode = asPropertyList.get(0).getValue();
         boolean typeSetsEnabled = getBoolean(platformValueNode, context, PlatformAttributes.ENABLED);
-        if (typeSetsEnabled == false) {
+        if (!typeSetsEnabled) {
```

Comment:
it's better to have PRs with code focused on a specific change that is advertised in the PR/commit message as a general best practice, however I'm not familiar with how PRs reviews work specifically in the agent repo.